packages/wallets/src/components/DerivAppsSection/DerivAppsGetAccount.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import DerivApps from '../../public/images/deriv-apps.svg';
+import { WalletButton, WalletText } from '../Base';
+
+type TProps = {
+    isMaltaWallet?: boolean;
+};
+
+export const DerivAppsGetAccount: React.FC<TProps> = ({ isMaltaWallet }) => {
+    return (
+        <div className='wallets-deriv-apps-section'>
```

Comment:
I tend to match BEM convention with component name, e.g. 
className="deriv-apps-get-account"
then later
className="deriv-apps-get-account__details"
(minor/optional)

Reply:
@wojciech-deriv As I know all classes in `wallets` package should start from `wallets-`. Next, `DerivAppsGetAccount` and `DerivAppsTradingAccount` are just part of the `DerivAppsSection` wrapper component and should not be used without this wrapper and these components use the same wrapper classes `wallets-deriv-apps-section__`